versus-fe/src/containers/LoginPage/LoginPage.js

```diff
@@ -33,21 +33,33 @@
 
     render() {
       return (
-        <div className="loginContainer">
-          <h1> Login </h1>
-          <div className="loginDetails">
-            <form onSubmit={this.sendDetails}>
-              Username <br/>
-              <input type="text" value={this.state.username} onChange={this.handleFormChange} name="username" />
-              <br/>
-              password <br/>
-              <input type="text" value={this.state.password} onChange={this.handleFormChange} name="password" />
-              <br/>
-              <input type="submit" value="Sign In"/>
-            </form>
+        <div className="background">
+
+          <div className="loginContainer">
+
+            <h1> Welcome to Versus </h1>
+
+            <div className="loginDetails">
+
+              <form onSubmit={this.sendDetails}>
+                <span>  Username  </span>
+                <input type="text" value={this.state.username} onChange={this.handleFormChange} name="username" />
+
+                <span>  Password </span>
+                <input type="text" value={this.state.password} onChange={this.handleFormChange} name="password" />
+              </form>
+
+            </div>
+
+            <div className="submit">
+              <text> Sign In </text>
+            </div>
+
+            <div className="Sign Up">
+              <a href="/SignUp"> Sign Up if you have do not have an account </a>
+            </div>
           </div>
-          <br/>
-          <a className="signUp" href=""> Sign up </a>
+
```

Comment:
Be careful with the spaces (blank lines) in html, I know sometimes it may feel condensed but in a 24 lines of html 12 are empty lines.

This is HTML not a Lays chips bag
![](https://www.fritolay.com/images/default-source/blue-bag-image/lays-classic.png?sfvrsn=bd1e563a_2)